### ISSUE TITLE
Translations of UI, table names and values

### ISSUE THREAD

**Issue by andreasneumann** [QGEP/QGEP#30]
* GUI forms and plugin GUI will be translated with QtLinguist
* table names and value lists will be automatically exported from the main database (through  Metadatabase) --> create a .ts file from the database
* Generate a view for export of the dictionary and the value lists

**Comment by m-kuhn** [comment]
That's how qwat does form translation: https://github.com/qwat/qWat/tree/master/qwat_dictionary

**Comment by m-kuhn** [comment]
@kfischerar takes care that field names are also translated in the database.
We will then probably generate a .qm translation file from the database.

**Comment by 3nids** [comment]
@andreasneumann wasn't it you that woud ask Stefan a translation table.

I would expect to get these columns

* table name
* field name
* field name in plain english
* field name in plain french
* field name in plain german
* field comment in plain english
* field comment in plain french
* field comment in plain german

Does it make sense?

**Comment by m-kuhn** [comment]
Database to .ts converter for database term translation.
For further translations, done on transifex https://www.transifex.com/projects/p/QGEP/
